lib/telegram/bot/client.rb

```diff
@@ -58,10 +58,10 @@ def extract_message(update)
       end
 
       def log_incoming_message(message)
-        uid = message.from ? message.from.id : nil
-        logger.info(
+        logger.info {
+          uid = message.try(:from) ? message.from.id : nil
```

Comment:
`try` is active-support method. The gem doesn't use active-support so this `try` won't work and produces an error.

That is why it was previously `message.from ? message.from.id : nil` and not `message.from.try(:id)`.